### ISSUE TITLE
About the estimated density map

### ISSUE THREAD

**Issue by liudakai2** [Cli98/DMNet#13]
Thank you for your excellent work. I have trouble with generating the estimated density map in both 'val' and 'test-dev' subset of VisDrone. All of the values in the estimated density maps of MCNN are below 0.08. In other words, no object-containing regions are activated. May you share your script for estimating density maps using MCNN?

And also, have you evaluated the performance on 'test-dev' subset? It seems that you only evaluated the 'val' subset in your paper, but the label of 'test-dev' subset is also available.

Thanks a lot.

**Comment by Mik030** [comment]
It seems like the sum of all  values in the window need to beyound 0.08.

Besides, I wonder the height and weight which are used in “Generate_density_map_official.py”. The author said in the paper that they used the average size of the objects in the training set. I hava analyze the Visdrone training set and computed the average size is 41*43, should I run the command "python density_slide_window_official.py . 41_43 0.08 --output_folder output_folder --mode val"? Thanks!

**Comment by Cli98** [comment]
> 

Hi @liudakai2 ,

Sorry for the late reply. density threshold (0.08 you referred) is an experimental value and thus may change between different dataset. Here you are not computing pixel threshold instead of the sliding window. Check @Mik030's comment. I am using visDrone 2018 dataset, thus yours may vary. You need to experiment for the optimal density threshold.

And we have not evaluated the performance on test-dev. By default test-dev dataset is only available on visdrone evaluation server. So we are not expected to report performance considering the fact that evaluation server is closed.

**Comment by Cli98** [comment]
> It seems like the sum of all values in the window need to beyound 0.08.
> 
> Besides, I wonder the height and weight which are used in “Generate_density_map_official.py”. The author said in the paper that they used the average size of the objects in the training set. I hava analyze the Visdrone training set and computed the average size is 41*43, should I run the command "python density_slide_window_official.py . 41_43 0.08 --output_folder output_folder --mode val"? Thanks!

@Mik030 your comment is correct. And the command to run is correct.

**Comment by Cli98** [comment]
Let me know if you have more questions.

**Comment by wwwweeerrrrttt** [comment]
> Let me know if you have more questions.

how can I experiment for the optimal density threshold?Run “Generate_density_map_official.py” with different density threshold?

**Comment by Cli98** [comment]
@wwwweeerrrrttt Hi and you are encouraged to try different values and visualized them after you generate crops via 8-connected components.

**Comment by Cli98** [comment]
Closed as no activity by far. Feel free to re-open for any further questions.
